datanode/sqlstore/migrations/0001_initial.sql

```diff
@@ -1260,8 +1260,7 @@ create table if not exists oracle_specs (
     id bytea not null,
     created_at timestamp with time zone not null,
     updated_at timestamp with time zone not null,
-    signers bytea[],
```

Comment:
We'll need some migrations for this I think

Reply:
This has been resolved, thanks!